lms/static/sass/views/_verification.scss

```diff
@@ -821,6 +829,104 @@
 
     // indiv slides - review
     #wrapper-review {
+      color: $black;
+
+      .page-title {
+          border-bottom: 2px solid $m-gray-d3;
+          padding-bottom: ($baseline*0.75);
+          margin-bottom: $baseline;
+          @extend %t-strong;
```

Comment:
@extend declarations should be at the top of the selector's styles (i.e. move it up to line 835)